src/ResourceManager/Compute/Stack/Commands.Compute.Test/ScenarioTests/ComputeTestCommon.ps1

```diff
@@ -408,13 +407,14 @@ function Create-ComputeVMImageObject
 # Get Default CRP Windows Image Object Offline
 function Get-DefaultCRPWindowsImageOffline
 {
-    return Create-ComputeVMImageObject 'MicrosoftWindowsServer' 'WindowsServer' '2008-R2-SP1' 'latest';
+    return Create-ComputeVMImageObject 'MicrosoftWindowsServer' 'WindowsServer' '2012-R2-Datacenter' '1.0.0';
 }
 
 # Get Default CRP Linux Image Object Offline
 function Get-DefaultCRPLinuxImageOffline
 {
-    return Create-ComputeVMImageObject 'SUSE' 'openSUSE' '13.2' 'latest';
+    #return Create-ComputeVMImageObject 'SUSE' 'openSUSE' '13.2' 'latest';
```

Comment:
@deepaft please remove this commented out line

Reply:
Removed.